org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -102,7 +154,14 @@ public void drawText(
    * @param endX the end X coordinate of the line.
    * @param endY the end Y coordinate of the line.
    */
-  public void drawLine(double startX, double startY, double endX, double endY) {}
+  public void drawLine(int startX, int startY, int endX, int endY) {
+    if (this.strokeColor == null) {
```

Comment:
Curious - any reason we allow drawing lines without a stroke color set but not other shapes?

Reply:
Yes, this was a request from curriculum that it would be a special case here, since lines only have strokes and not fills.